tests/test_pipeline.py

```diff
@@ -0,0 +1,86 @@
+from typing import List
+
+import pytest
+import spacy
+from spacy.tokens import Span
+from spacy.util import filter_spans
+
+from replacy import ReplaceMatcher
+from replacy.suggestion import Suggestion
+
+nlp = spacy.load("en_core_web_sm")
```

Comment:
I wonder ... we allow replacy to use any nlp, but we require en_core_web_sm in tests? :grimacing: 
It is ok?

Reply:
We should add tests with other languages!